tags.yaml

```diff
@@ -70,6 +70,7 @@
 - government spending
 - health
 - high-throughput imaging
+- hindcast
```

Comment:
To be fair, our tagging structure is not yet well defined, but I feel like a lot of these tags are either too specific or too generic. If you'd like to help us get the tagging right for a subset of the datasets, we'd be open to that! But in the absence of looking across how to do this better for a number of the datasets, I'll ask that you remove these since they're only being added to one dataset. Thanks!

Reply:
Will do